hooks/post_gen_project.py

```diff
@@ -18,6 +18,12 @@ def main():
         cli_file = os.path.join('{{ cookiecutter.project_slug }}', 'cli.py')
         remove_file(cli_file)
 
+    if '{{ cookiecutter.ci_provider }}' != 'Travis CI':
+        remove_file('.travis.yml')
+    elif '{{ cookiecutter.ci_provider }}' != 'Github Actions':
```

Comment:
I think this should be an `if` rather than `elif`. Otherwise, the `No continuous integration` option will enter the previous fork and then skip this.

Reply:
oh good point, fixed